modules/virt-mounting-windows-driver-disk-on-vm.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-edit-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-mounting-windows-driver-disk-on-vm_{context}"]
+
+= Mounting a Windows driver disk on a virtual machine
+
+You can mount a Windows driver disk on a virtual machine (VM) by using the {product-title} web console.
+
+.Procedure
+
+. Navigate to *Virtualization* -> *VirtualMachines*.
+. Select the required VM.
++
+The *VirtualMachine details* page opens.
+. On the *Configuration* tab, click *Storage*.
+. Select the *Mount Windows drivers disk* checkbox.
++
+The Windows driver disk appears in the list of mounted disks.
```

Comment:
This is a global comment for the affected files: perhaps you could check if it should be a `driver disk for Microsoft Windows` rather than `Windows driver disk`.

Reply:
@max-cx It's not a disk FOR drivers, it's a disk that contains the drivers. As for Microsoft, I am not sure - in our documentation, Microsoft is almost never mentioned.